Apptentive/Apptentive/Apptentive.h

```diff
@@ -778,6 +788,8 @@ typedef NS_ENUM(NSUInteger, ApptentiveLogLevel) {
 
 @property (assign, nonatomic) ApptentiveLogLevel logLevel;
 
+@property (assign, nonatomic) BOOL redactSensitiveInformation;
```

Comment:
@frankus, is this property used anywhere?

Reply:
No, it doesn't appear to be.